erts/emulator/test/bs_construct_SUITE.erl

```diff
@@ -520,6 +520,14 @@ do_more(Bin, Sz) ->
 do_something() ->
     throw(blurf).
 
+append_empty_is_same(Config) when is_list(Config) ->
```

Comment:
Can you add a test for when a writeable binary is appended to?

Reply:
I would love to except I have no idea how to write it. I didn't even know writable binaries were a thing before this PR. Can you provide a quick snippet? Thanks!

Reply:
Maybe take a look at what the copy_writeable_binary test does? i.e.

    WriteableBin = <<(id(<<>>))/binary,0,1,2,3,4,5,6,7>>,

Reply:
Thanks! I have added an assertion and verified that the assertion fails without the proper fix in the writeable branch of the code.